File: app/src/main/java/com/sourabh/android/rxretrorealm/modal/Repo.java
package com.sourabh.android.rxretrorealm.modal;

import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

public class Repo extends RealmObject {
    @SerializedName("id")
    private String id;
    @SerializedName("name")
    private String name;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/sourabh/android/rxretrorealm/modal/Repo.kt
package com.sourabh.android.rxretrorealm.modal

import com.google.gson.annotations.SerializedName

import io.realm.RealmObject

public open class Repo:RealmObject(){
    @SerializedName("id")
    public open var id: String? = null
    @SerializedName("name")
    public open var name: String? = null
}
